src/pages/Accounts.vue

```diff
@@ -103,9 +134,17 @@ export default {
     this.$store.dispatch('updateAllBalances')
   },
   methods: {
-    ...mapMutations(['selectIdentity', 'toggleIdManager', 'createIdentity']),
+    ...mapMutations(['selectIdentity', 'toggleIdManager', 'createIdentity', 'setName']),
     activateCard (i) {
       this.activeIdentityCard = i === this.activeIdentityCard ? -1 : i
+    },
+    handleAddAddress () {
+      let accountCount = this.$store.state.mobile.accountCount
+      let addedName = this.addedName
+      this.createIdentity()
+      this.setName({objIndex: accountCount, name: addedName})
```

Comment:
`createIdentity` is called only in this component, so you should pass the name of a new account directly to this mutation instead of `setName`.